src/screens/SendConfirmationSheet.tsx

```diff
@@ -526,6 +525,15 @@ export default function SendConfirmationSheet() {
     shouldShowChecks,
   });
 
+  let message;
```

Comment:
very nit but I prefer these in a lil fn so its incapsulated 

getMessage = () {
..blah blah
return message
}